bgpd/bgp_network.c

```diff
@@ -625,6 +625,8 @@ int bgp_getsockname(struct peer *peer)
 			"%s: nexthop_set failed, resetting connection - intf %p",
 			peer->host, peer->nexthop.ifp);
 		return -1;
+#else
+		zlog_info("%s: nexthop_set failed some functionality may be unavaliable", peer->host);
```

Comment:
Can we use the latest flog_err() ?

Reply:
This is a zlog_info?  I left this as informational because the non-cumulus code path doesn't treat it as an error, so I want to note the fact this happened.